oak-auth-external/src/test/java/org/apache/jackrabbit/oak/spi/security/authentication/external/basic/DefaultSyncContextTest.java

```diff
@@ -1201,6 +1201,36 @@ public void testSyncPropertiesRemapped() throws Exception {
         }
     }
 
+    @Test
```

Comment:
minor wish: i don't do super consistently either but having a javadoc above the test inciting for which JIRA ticket this test was added is usually helpful if one needs to investigate later on.
since a dedicated jira ticket is anyway needed for the commit message you may want to add it as a comment here at the same time.

Reply:
thanks!